modules/ROOT/pages/play-share-api-notebook-task.adoc

```diff
@@ -12,26 +12,26 @@ After creating an API Notebook, you play the notebook as described in this proce
 +
 The Developer portal for the T-Shirt Ordering Service appears.
 +
-image::tutorial-create-an-api-notebook-993e9.png[]
+image::tutorial-create-an-api-notebook-993e9.png["API Notebook interface showing code editor and execution controls"]
 +
 . From the bottom of the API Reference on the portal, copy the root RAML URL.
 +
-image::rootraml.png[height=224,width=797]
+image::rootraml.png["RAML specification editor with root level configuration options",height=224,width=797]
 . Select the API Notebook from the list of visible components.
 . In API.createClient, check that the URL matches the root RAML URL and replace if necessary.
 . Click Play notebook to execute the Notebook code. The results reveal information about your API based on the RAML.
 . On the Developer portal for the API, at the bottom of a published API Notebook, click Make your Own to save the API Notebook to Github as a gist.
 +
 A copy of the Notebook appears.
 +
-. Click image:save.png[].
+. Click Save image:save.png[""].
```

Comment:
```suggestion
. Click *Save* image:save.png[""].
```

Reply:
Not sure if it's a UI element. I added the word for blind people.